mLRS/CommonTx/cli.h

```diff
@@ -334,6 +340,10 @@ class tTxCli
     uint8_t task_pending;
     int32_t task_value;
 
+    CLI_OUTPUT_ENUM output_pending;
+    int16_t output_index;
```

Comment:
maybe print_chunk_idx, more descriptie IMHO and not so easily confused with the param idx, it's also a bit more along what we use in mavlink

Reply:
In the case of help, it is indeed a chunk index, but in the case of pl, it actually is a parameter index!  In both cases, it's part of our print state and if we are short of RAM, I'd rather not use more of it.  I think I'll call it print index.